.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: If you can think of an improvement to a project then tell us about it!
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Is this a new feature or an improvement to something that already exists?
+Select 1 checkbox
+- [ ] New feature
+- [ ] Improvement
```

Comment:
This should be a label, not a checklist. 1) Excessive checklists in issue templates are the bane of readability, and 2) checklists aren't for mutually exclusive items. Use an issue form instead for something like this: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms